packages/manager/modules/emailpro/src/domain/dkim-autoconfig/emailpro-domain-dkim-autoconfig.controller.js

```diff
@@ -20,6 +21,7 @@ export default class EmailProDomainDkimAutoconfigCtrl extends DkimAutoConfigurat
       $scope,
       EmailProDomains,
       navigation,
+      messaging,
```

Comment:
Seems to be unused. Why that service is injected ?

Reply:
Yes it is used, in the parent classe

Reply:
![image](https://github.com/ovh/manager/assets/8767496/6dcc1e39-dfd3-4ab0-8c04-4bf43497fedd)